forms-flow-api/src/formsflow_api/services/external/keycloak.py

```diff
@@ -42,7 +42,8 @@ def __init__(self):
             }
         )
         self.base_url = (
-            f"{current_app.config.get('KEYCLOAK_URL')}/auth/admin/realms/"
+            f"{current_app.config.get('KEYCLOAK_URL')}/"
+            f"{current_app.config.get('KEYCLOAK_URL_HTTP_RELATIVE_PATH', 'auth/')}/admin/realms/"
```

Comment:
Here if no value is provided, wouldn't it cause 2 slashes ? i.e, `auth//admin/realms`. Please fix it

Reply:
Indeed it would. I made the change too hastily it seems, will fix